test/catalog.tests.bom

```diff
@@ -22,6 +22,11 @@ brooklyn.catalog:
         launch.command: |
           while true ; do sleep 3600; done & echo $! > ${PID_FILE}
         checkRunning.command: ""
+        brooklyn.config:
+          pre.install.command:
+            sudo yum install -y yum-plugin-fastestmirror
+            sudo yum update -y
+            sudo yum -y install python27
```

Comment:
Should we do this (i.e. install python27) in the cassandra-node entity, rather than just in the test code?